codenames-domain/src/main/java/com/codenames/mapper/PlayerMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.codenames.mapper;
+
+import com.codenames.dto.UserDto;
+import com.codenames.models.game.Player;
+import org.mapstruct.IterableMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
+
+@Mapper
+public interface PlayerMapper {
+
+    PlayerMapper INSTANCE = Mappers.getMapper(PlayerMapper.class);
```

Comment:
I am not sure why exactly do you need to instantiate mapper instance each time. It should work without that.

Reply:
If that's the case, remove it from all of your mappers

Reply:
Yes, it will work without INSTANCE, it can just be used to avoid getting the mapper itself